third_party/docfx/templates/devsite/partials/uref/class.header.tmpl.partial

```diff
@@ -46,6 +46,62 @@
 <h2>{{__global.remarks}}</h2>
 <div class="markdown level0 remarks">{{{remarks}}}</div>
 {{/remarks}}
+{{#inherits.0}}
+<div class="inheritance">

```

Comment:
There is already an inheritance section above. Can we reuse it? This appears to be functionally the same, so I'd prefer to have consistent style between languages and a simpler template.